jsx/Form.js

```diff
@@ -1428,25 +1458,33 @@ class CheckboxElement extends React.Component {
     // Add error message
     if (this.props.errorMessage) {
       errorMessage = <span>{this.props.errorMessage}</span>;
-      elementClass = 'checkbox-inline col-sm-offset-3 has-error';
+      elementClass = this.props.class = 'checkbox-inline'
```

Comment:
this still doesn't look right. i think it should actually just be `elementClass = elementClass + ' has-error'` no?

Reply:
I messed up there pretty bad. Thanks @zaliqarosli !